BridgeSDK/BridgeAPI/SBBActivityManager.m

```diff
@@ -56,13 +58,80 @@ + (instancetype)defaultComponent
     return shared;
 }
 
-- (NSURLSessionDataTask *)getScheduledActivitiesForDaysAhead:(NSInteger)daysAhead withCompletion:(SBBActivityManagerGetCompletionBlock)completion
+- (NSURLSessionDataTask *)getScheduledActivitiesForDaysAhead:(NSInteger)daysAhead withCompletion:(SBBActivityManagerGetCompletionBlock)completion {
+    SBBCachingPolicy policy = gSBBUseCache ? SBBCachingPolicyFallBackToCached : SBBCachingPolicyNoCaching;
+    return [self getScheduledActivitiesForDaysAhead:daysAhead cachingPolicy:policy withCompletion:completion];
+}
+
+- (NSURLSessionDataTask *)getScheduledActivitiesForDaysAhead:(NSInteger)daysAhead cachingPolicy:(SBBCachingPolicy)policy withCompletion:(SBBActivityManagerGetCompletionBlock)completion
+{
+    return [self getScheduledActivitiesForDaysAhead:daysAhead daysBehind:0 cachingPolicy:policy withCompletion:completion];
```

Comment:
I was thinking it didn't have to be backward-compatible since anyone using BridgeSDK currently is handling this on their own (e.g. in AppCore), so if they refactored to take advantage of caching they'd explicitly state how many daysBehind they want anyway. But yeah, if the default is 1 most AppCore apps wouldn't actually have to update this call and they'd still get what they need, so that's better. I'll change it.